lib/charms/observability_libs/v1/kubernetes_service_patch.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
so, what's the story here?  why are we copy/pasta libraries that can't be pip installed?

Reply:
this is a charm library brought in with `charmcraft fetch-lib charms.observability_libs.v0.kubernetes_service_patch`. I too feel this is an odd way to handle a library dependency, but I guess this is the way the framework does things.

Reply:
🤮